modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,275 @@
+... Select the *Copy Link* icon next to *Installer package* to copy the code snippetL, as shown in the following example:
++
 ```
 rtfctl appliance upgrade --url https://<upgrade-package-information>
 ```
+... Paste the URL in a text editor for reference. 
+.. Start the upgrade procedure:
+... Using your terminal open on a controller node, run the following command, substituting the value of the URL parameter with the *Installer package* URL: 
++
+----
+sudo ./rtfctl appliance upgrade --url <cluster-installer-url>
+----
++
+This command also supports referencing an installer package already downloaded on the node, with the `--file` parameter.
++
+The command outputs a confirmation indicating the upgrade is being performed in the background.
++
+.. Run `sudo gravity status` on a node and verify that the cluster status is “updating”.
+.. Follow the progress on the upgrade procedure:
+... Using your terminal open on a controller node, run the following command: 
++
+----
+sudo gravity plan
+----
++
+.. Confirm the upgrade has completed successfully:
+... Run `sudo gravity status` and verify that the cluster status transitioned from “updating” to “active”.
+.. If the Runtime Fabric cluster version was 1.0.x prior to upgrading, and an HTTP proxy is in use, run this command to apply the HTTP proxy settings: 
++
+----
+sudo ./rtfctl apply http-proxy --confirm existing
+----
+.. Perform the following step on *every node* to make sure system configurations are up to date:
 
-[IMPORTANT]
-To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, upgrade to 1.4.54 before upgrading to 1.5.
+... Open a terminal to your Runtime Fabric controller/worker node.
+... If needed, download the latest `rtfctl` command-line utility:
++
+----
+cd /opt/anypoint/runtimefabric
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
++
+... Change file permissions for the `rtfctl` binary: 
++
+----
+chmod +x rtfctl
+----
++
+... Run the `apply system-configurations` command in `rtfctl`:
++
+----
+sudo ./rtfctl apply system-configuration 
+----
+
+== Resume an Upgrade
+
+If the cluster-level (installer package) components upgrade procedure encountered a failed step, try to resume the upgrade. 
+
+Resumed upgrades are attached to your terminal session. Ensure you have a stable connection before attempting to resume an upgrade.
+
+. On a terminal open to the controller node that was used to start the upgrade, change to the directory with the installer bundle files, as shown in the following example:
++
+----
+cd /opt/anypoint/runtimefabric/installer
+----
++
+. Run the command to resume the upgrade: 
++
+----
+sudo ./gravity upgrade --resume
+----
++
+. The upgrade continues streaming output to your terminal session. 
+
+If the error occurs again, follow the troubleshooting steps described in the following section.
+ 
+== Troubleshooting Upgrade Errors
```

Comment:
I think troubleshooting steps would be based upon which upgrade was taken (which component was being upgraded).

Reply:
I added a clarification for this.